LeadAlerts.Web/Domain/MessageSender.cs

```diff
@@ -1,28 +1,21 @@
 ﻿using Twilio;
+using Twilio.Types;
+using Twilio.Rest.Api.V2010.Account;
+using System.Threading.Tasks;
 
 namespace LeadAlerts.Web.Domain
 {
-    public interface IMessageSender
```

Comment:
I think you should keep an interface for MessageSender, it will be easier to mock